java/google/registry/config/ConfigModule.java

```diff
@@ -784,4 +784,14 @@ public static int provideMaxChecks(RegistryConfig config) {
   public static Duration getAsyncDeleteFlowMapreduceDelay() {
     return Duration.standardSeconds(90);
   }
+
+  /**
+   * Per RFC {@link: https://tools.ietf.org/html/rfc5730} 'svID' is the element that
+   * contains the name of the server
+   */
+  @Provides
+  @Config("svID")
```

Comment:
On its own, I think "svID" might be a little too ambiguous.  Maybe we could call this "greetingSvID" or "greetingServerId" or "greetingServerName" to make it clear that it's just about setting what's used in the EPP greeting?